hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/OzoneContainer.java

```diff
@@ -160,8 +160,12 @@ private void startContainerScrub() {
       LOG.info("Background container scrubber has been disabled by {}",
               HddsConfigKeys.HDDS_CONTAINERSCRUB_ENABLED);
     } else {
-      this.scrubber = new ContainerScrubber(containerSet, config);
-      scrubber.up();
+      if (this.scrubber == null) {
+        this.scrubber = new ContainerScrubber(containerSet, config);
+      }
+      if (this.scrubber.isHalted()) {
```

Comment:
I think we don't need this check here, as scrubber up() is already taking care of multiple starts.
public void up() {

    this.halt = false;
    if (this.scrubThread == null) {
      this.scrubThread = new Thread(this);
      scrubThread.start();
    } else {
      LOG.info("Scrubber up called multiple times. Scrub thread already up.");
    }
  }